pkgs/servers/spicedb/zed.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "zed";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "authzed";
+    repo = "zed";
+    rev = "v${version}";
+    hash = "sha256-tw8Z8JtmmRLcvFacRDAdIi6TyMtm9FAZvRYNgd49qXg=";
+  };
+
+  vendorHash = "sha256-/BxQiaBFkJsySnQRU870CzvPxtPwvdwx4DwSzhaYYYQ=";
+
+  meta = with lib; {
+    description = "Command line for managing SpiceDB.";
```

```suggestion
    description = "Command line for managing SpiceDB";
```
No need to include a period in the end.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description